config/index.js

```diff
@@ -1,2 +1,2 @@
-import { View, ViewPropTypes as RNViewPropTypes } from 'react-native';
-export const ViewPropTypes = RNViewPropTypes || View.propTypes;
+import PropTypes from 'prop-types';
+export const ViewPropTypes = PropTypes;
```

Comment:
This won't work since `Collapsible` expects this `ViewPropTypes` variable to have a `style` property. Instead you can just delete the entire `config` folder, delete the import on line 4 of `Collapsible.tsx` and change `ViewPropTypes.style` to `PropTypes.object` on line 16. Same thing for `Accordion.tsx`.

Reply:
Thank you so much for the review 😄